style.css

```diff
@@ -3,22 +3,33 @@
 body {
   min-height: 100%;
   font-family: 'Poppins', sans-serif;
+  background-color: #e5e5e5;
 }
 
-@media only screen and (min-width: 768px) {}
+@media only screen and (min-width: 768px) {
+  /* stylelint-disable-line block-no-empty */
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.container {
+  display: grid;
+  place-items: center;
+}
```

Comment:
- [ ] **[OPTIONAL]** I think it would be better to prevent scrolling horizontally in your CSS for the best user experience and also to be able to design your webpage to replicate the Figma design 👍 👍 

example:
```
body {
  overflow-x: hidden;
  width: 100%;
}
```
_you may kindly follow this[ link ](https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp)for more info on hidden overflow_